src/util/getReceiverAddress.ts

```diff
@@ -46,15 +46,17 @@ const getReceiverAddress = ({ targetChainDetails, sourceChainDetails, receiverAd
   };
 
   try {
-    const [validatedDerivedAcccount, rest] = checkAddress(receiverAddress, targetSS58Format);
+    const decodedReceiverAddress = base58Decode(receiverAddress);
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const [isValidDerivedAcccount, _length, _ss58DecodedLength, ss58Decoded] = checkAddressChecksum(
```

Comment:
i much prefer your initial approach of empty values rather than escaping this that shows the warning in the console.

Reply:
Haha me too! Wasn't sure what to do with the comment earlier, is just one of those things that sucks in JS I suppose 😄